dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -156,6 +156,14 @@ JSON_FIELDS_INVALID_CHARACHTER=TRAS3011W: An invalid character was provided in t
 JSON_FIELDS_INVALID_CHARACHTER.explanation= Only letters, numbers, and : are accepted in the JSON logging field name mapping property
 JSON_FIELDS_INVALID_CHARACHTER.useraction= Change your JSON logging field name mapping property value to remove invalid characters
 
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING=TRAS3012W: The log rollover start time property is incorrectly formatted. The value will be set to a default of '00:00', representing midnight.
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING.explanation=The rollover start time property must be formatted HH:MM, with 00:00 representing midnight.
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING.useraction=The rollover start time property must be formatted HH:MM, with 00:00 representing midnight.
+
+LOG_ROLLOVER_INTERVAL_TOO_SHORT_WARNING=TRAS3013W: The log rollover interval cannot be less than 1 minute long. The value will be set to a default of 1 day.
+LOG_ROLLOVER_INTERVAL_TOO_SHORT_WARNING.explanation=TRAS3013W: The log rollover interval cannot be less than 1 minute long.
+LOG_ROLLOVER_INTERVAL_TOO_SHORT_WARNING.useraction=TRAS3013W: Please provide a log rollover interval equal to or greater than 1 minute.
```

Comment:
Please provide a log rollover interval equal to or greater than 1 minute.
--->
Specify a log rollover interval that is equal to or greater than 1 minute.